Src/Plugins/Tests.Security/Validators/SEC101_005.SlackTokenValidatorTests.cs

```diff
@@ -30,5 +35,106 @@ public void SlackTokenValidator_Test()
+                                                    "\"user\": \"testbot\"," +
+                                                    "\"team_id\": \"1234ABCD\"," +
+                                                    "\"user_id\": \"5678EFGH\"," +
+                                                    "\"bot_id\": \"0987ZYXV\"," +
+                                                    "\"is_enterprise_install\": false}",
+                                                    Encoding.UTF8,
+                                                    "application/json").As<HttpContent>(),
+                    ExpectedValidationState = ValidationState.Authorized,
+                    ExpectedMessage = "Bot token (id: 0987ZYXV) was authenticated to channel 'testteam.slack.com',  team 'test team' (id: 1234ABCD),  user 'testbot' (id: 5678EFGH).  The token is not associated with an enterprise installation."
+                },
+                new
+                {
+                    Title = "Testing Invalid Token",
+                    HttpStatusCode = HttpStatusCode.OK,
+                    HttpContent = new StringContent("{\"ok\": true," +
+                                                    "\"error\": \"invalid_auth\"}",
+                                                    Encoding.UTF8,
+                                                    "application/json").As<HttpContent>(),
+                    ExpectedValidationState = ValidationState.Unauthorized,
+                    ExpectedMessage = "The provided secret is not authorized to access 'xoxb-1234'."
+                },
+                new
+                {
+                    Title = "Testing Revoked Token",
+                    HttpStatusCode = HttpStatusCode.OK,
+                    HttpContent = new StringContent("{\"ok\": true," +
+                                                    "\"error\": \"token_revoked\"}",
+                                                     Encoding.UTF8,
+                                                     "application/json").As<HttpContent>(),
+                    ExpectedValidationState = ValidationState.Expired,
+                    ExpectedMessage = string.Empty
+                },
+                new
+                {
+                    Title = "Testing Inactive Token",
+                    HttpStatusCode = HttpStatusCode.OK,
+                    HttpContent = new StringContent("{\"ok\": true," +
+                                                    "\"error\": \"account_inactive\"}",
+                                                    Encoding.UTF8,
+                                                    "application/json").As<HttpContent>(),
+                    ExpectedValidationState = ValidationState.Expired,
+                    ExpectedMessage = string.Empty
+                },
+                new
+                {
+                    Title = "Testing Unknown Slack Error",
+                    HttpStatusCode = HttpStatusCode.OK,
```

Comment:
>HttpStatusCode.OK [](http://example.com/codeflow?start=37&length=17)

let's add badrequest ->  httpcontent = null -> and save as unexpectedstatuscode

Reply:
done.